src/renderer/components/basic.lua

```diff
@@ -22,6 +22,23 @@ local function calcSizeBig(text)
   return math.ceil(calcWidth(text) / 2) * 2, math.ceil(font.height / 3) * 3
 end
 
+local function calcTextSize(node)
+  if not node.styles["font-size"] then
+    return 1
```

Comment:
If an element doesn't have an explicit `font-size`, shouldn't it inherit its parent's size?

Reply:
Ah, right!